### ISSUE TITLE
[Screenreader] While typing into input, after each character an error is read

### ISSUE THREAD

**Issue by micahchiang** [department-of-veterans-affairs/formulate#88]
## Description
[Original DST ticket](https://app.zenhub.com/workspaces/vsp-design-system-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/va.gov-team/5875)

While typing numbers into the input, after each character is read by the screen reader, an error is read. (See original ticket for exampl)

## Acceptance criteria
As a screen reader user, I want to enter values in the input and not experience an error notification until my input is complete.

## Steps to recreate
1. Visit staging.va.gov/profile/
2. Start screenreader (JAWS)
3. Navigate to edit the mailing address
4. Navigate to the zip code input
5. Delete the value and begin typing a new zip code
6. Verify the validation begins while the focus is on the input and only a few characters entered

**Comment by sdk-irl** [comment]
@micahchiang I've temporarily unassigned myself to this one since we're still getting the screen reader stuff worked out, and I started taking a look at #89 today

**Comment by taylorkaren** [comment]
Previous Slack conversation looking for advice on how to test with JAWS or NVDA: https://dsva.slack.com/archives/C8E985R32/p1642017844001800. This will need to be resolved if we pick up work on this ticket again.

**Comment by taylorkaren** [comment]
Similar to [Ticket #529](https://github.com/department-of-veterans-affairs/va.gov-team/issues/529). Per entry on the [Forms Library Accessibility Issues Super Set List](https://docs.google.com/spreadsheets/d/1M9wfqjLNjecPvwhl8XmJxBJi2CbiOyRS-_9uI3Dn678/edit#gid=00, Ticket #529, is now closed. Matt is working on getting help from an accessibility specialist to retest this ticket to see if it is still something that needs to be addressed.

**Comment by taylorkaren** [comment]
@CourtneyChambers77 retested this issue on the original ticket (https://github.com/department-of-veterans-affairs/va.gov-team/issues/5875#issuecomment-1120088250) and it's still an issue so we will need to look into this.
